api/health/health.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package health;
+
+option go_package = "github.com/talos-systems/talos/api/health";
+
+import "google/protobuf/empty.proto";
+
+service Health {
+  rpc Check(google.protobuf.Empty) returns (HealthCheckResponse);
+  rpc Watch(HealthWatchRequest)  returns (stream HealthCheckResponse);
+  rpc Ready(google.protobuf.Empty) returns (ReadyCheckResponse);
+}
+
+message HealthWatchRequest {
+  int64 interval_seconds = 1;
```

Comment:
No more count?

Reply:
not necessary as you pointed out, this will work. if we need add count later it'd be an additive change which should be easy enough to implement after the fact.

Reply:
Might even be something we leave up to client side implementations.